### PR TITLE
feat(BaseInput): add readonly style

### PR DESCRIPTION
**Pull request by LucidityDesign**
I disabled the focus state when the input field is set to read only.

### CODE REVIEW

File: src/components/controls/inputs/BaseInput/BaseInput.tsx [hunk 1]
```diff
@@ -126,12 +127,18 @@ const StyledInput = styled.input<StyledInputProps>`
   ${({ leftIcon, theme }) => leftIcon && `padding-inline-start: ${4.5 * theme.spacingBase}rem`};
   ${({ rightIcon, theme }) => rightIcon && `padding-inline-end: ${6 * theme.spacingBase}rem`};
 
-  &:focus {
+  &[readonly] {
+    padding-inline-start: 0;
+    background: none;
+    outline: 0;
```
Comment (filiptammergard) [inline src/components/controls/inputs/BaseInput/BaseInput.tsx:133]: I think we need to go through this with Rui. We probably need some kind of readonly+focus state.

File: src/components/controls/inputs/TextInput/TextInput.stories.tsx [hunk 2]
```diff
@@ -41,6 +41,18 @@ WithoutLabel.play = async ({ canvasElement }) => {
   await expect(input).not.toHaveErrorMessage()
 }
 
+export const ReadOnly = Template.bind({})
+ReadOnly.args = {
+  label: "Label",
+  value: "Lorem Ipsum",
```
Comment (filiptammergard) [inline src/components/controls/inputs/TextInput/TextInput.stories.tsx:47]: Maybe "Read-only value", to match with the value of the DefaultValue story?

**Comment by filiptammergard** [comment]
Feel free to re-request review when readonly+focus state is there!

**Comment by LucidityDesign** [comment]
Styling has been approved by Rui

**Review by filiptammergard — APPROVED** [review]
Do you get why the test-storybook check is failing?

**Pull request merged**